source/sitemap.xml.haml

```diff
@@ -7,3 +7,5 @@
     - next if resource.path =~ /^ui-kit/
     %url
       %loc= "#{base_url}#{resource.url}"
+  %url
+    %loc = "#{base_url}/documentation/sitemap.xml"
```

Comment:
Do we want to reference the sitemap here? My understanding was that it's appropriate to reference other sitemap files within a `sitemapindex` [document](https://www.sitemaps.org/protocol.html#index), but I'm not sure what the effect, if any, of including a sitemap reference in a `sitemap` document.

My understanding of the spec is that we can [make search engines aware](https://www.sitemaps.org/protocol.html#submit_robots) of both /sitemap.xml and the new /documentation/sitemap.xml by either:

* Adding an additional line to robots.txt: `"Sitemap: #{base_url}/documentation/sitemap.xml"`
* Creating a new sitemap-index.xml file, a `sitemapindex` XML document which references both sitemaps, and then update robots.txt to reference only sitemap-index.xml